install.sh

```diff
@@ -133,22 +136,44 @@ download_vip() {
 }
 
 main() {
-  #validate arguments
-  if [[ "${#}" -ne 0 ]]; then
-    >&2 echo "error: invalid arguments '${*}', see 'bash ${SCRIPT_NAME} --help' for more information"
-    exit 1
-  fi
+  local -r args=$(getopt -a -n install -o p,h --long prune,help -- "$@")
```

Comment:
current install instructions
```
curl -sSL https://download.molgeniscloud.org/downloads/vip/install.sh | bash
```
do not allow using the prune feature and are inconsistent with the 'set all options via environment variables instead of command-line arguments'.

I think we should support one of these options:
1. add environment variable VIP_PRUNE, do not use getopt
2. add cli args as second option next to environment variables and add options for all environment variables with cli args taking precedence over environment variables

How about 2.?